regression/testdata/12hr80tps4org2chan-network-spec.yml

```diff
@@ -91,17 +93,19 @@ applicationCapabilities: V1_4_2
 numChannels: 2
 
 k8s:
+  namespace: fabric-system-test
   serviceType: NodePort
```

Comment:
I thought @btl5037 said we were no longer using NodePort anywhere.

Reply:
We still use NodePort, it's just not part of the YTT templates anymore. It calculated at runtime now

Reply:
this is service type. He referred IP address of worker node when selected NodePort as service type.